experiment/coverage_utils.py

```diff
@@ -137,10 +174,36 @@ def merge_profdata_files(self):
     def generate_coverage_summary_json(self):
         """Generates the coverage summary json from merged profdata file."""
         coverage_binary = get_coverage_binary(self.benchmark)
+
```

Comment:
Why are changes to this function needed?

Reply:
This function has been changed to produce the merged json summary (as before) and trial-specific json summary from trial-specific profdata so that information particular to one trial for any benchmark-fuzzer can be recorded as well.